rector.php

```diff
@@ -0,0 +1,32 @@
+<?php
+use Rector\Config\RectorConfig;
+use Rector\TypeDeclaration\Rector\Property\TypedPropertyFromStrictConstructorRector;
+use Rector\Php81\Rector\FuncCall\NullToStrictStringFuncCallArgRector;
+use Rector\Php81\Rector\Property\ReadOnlyPropertyRector;
+
+return RectorConfig::configure()
+    ->withPaths([ __DIR__ . '/' ])
+    ->withSkip([
```

Comment:
Would it be easier to just point rector at `src` and `tests`, and ignore everything else? We shouldn't have any source files in other folders.